models/metrics/erc1155_mint_burn_count.sql

```diff
@@ -0,0 +1,50 @@
+{{ 
+    config(
+        materialized='table',
+        tags=['metrics'],
+        cluster_by=['day_date']
+    )
+}}
+
+with events as (
+select
+    block_timestamp,
+    ingested_at,
+    event_name,
```

Comment:
should also need the evm_address of which contract that is being burnt or minted